x-pack/filebeat/input/awss3/config_test.go

```diff
@@ -408,19 +408,6 @@ func TestConfig(t *testing.T) {
 			expectedErr: "path_style can only be used when polling non-AWS S3 services",
 			expectedCfg: nil,
 		},
-		{
-			name:           "error on path_style with AWS SQS Queue",
```

Comment:
can we put the deleted test back?

Reply:
This test criteria has changed now as we added an extra condition which says it is not an error if `QueueURL` is present. I have rolled back other test where `PathStyle` is `true` , `QueueURL` is `""` , `NonAWSBucketName ` is `""`